csi/volumeutils.py

```diff
@@ -1135,6 +1135,19 @@ def mount_glusterfs_with_host(volname, mountpoint, hosts, options=None, is_clien
     if not os.path.exists(mountpoint):
         makedirs(mountpoint)
 
+    # FIXME: make this better later (an issue for external contribution)
+    # opt_array = None
+    # if options:
+    #     opt_array = []
+    #     for opt in options.split(","):
+    #         if not opt or opt == "":
```

Comment:
Use `opt.strip()` to compare as empty string. (Example: `-o "log-level= DEBUG,no-root-squash, "`)